babel.config.js

```diff
@@ -4,4 +4,5 @@ module.exports = {
     '@babel/preset-typescript',
     '@babel/preset-react',
   ],
+  plugins: [['@babel/transform-runtime']],
```

Comment:
What's this for?

Reply:
Removed. I'd like to keep babel config simple for now so I've opted to not add in `Async Await` syntax.
Whilst updating babel is possible I think it could cause issues for consumers.